src/main/resources/userEmailTemplate-feedbackSessionsAccessLinks.html

```diff
@@ -0,0 +1,24 @@
+<p>Hello ${userName},</p>
+
+<p>
+  You are receiving this email because someone requested <a href="${teammateHomePageLink}">TEAMMATES</a> (an online peer evaluation system) to send access links to this email address. If you did not make that request, please ignore this email.
+</p>
+
+<p>
+  Below are the feedback sessions that you participated in for the recent six month:
```

Comment:
months

Reply:
Also, standardise whether to spell out `six` or use the digit `6` (currently, both forms are used in the email).